.github/workflows/tests.yml

```diff
@@ -7,23 +7,24 @@ on:
 jobs:
   rake:
     runs-on: "ubuntu-latest"
-    continue-on-error: true
```

Comment:
Allow the build to fail, if the tests don't pass for any of the supported Rubies.

Reply:
👍